core/src/test/java/org/elasticsearch/http/netty/NettyHttpChannelTests.java

```diff
@@ -169,16 +175,54 @@ public void testHeadersSet() {
         assertThat(response.headers().get(HttpHeaders.Names.CONTENT_TYPE), equalTo(resp.contentType()));
     }
 
+    public void testLimiterAbortsTooLargeRequests() {
```

Comment:
I think the test needs to be renamed? also what are we testing here? it seems we call the breaking directly, so we're actually testing the breaker?